package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "node_modules/.bin/_mocha  $(find . -name '*.spec.js')",
-    "test:lint": "node_modules/eslint/bin/eslint.js ./  || exit 0",
+    "test:lint": "eslint .",
```

Comment:
Why do you make it exit with 0 even if it fails? I can re add it, seems weird, though 😄